icons/this file only exists to get github to make a folder

```diff
@@ -0,0 +1 @@
+Rucker#1337 was here
```

Comment:
```suggestion
<!--
SPDX-FileCopyrightText: "Year" "Nickname or Name"
SPDX-License-Identifier: CC0-1.0
-->
Rucker#1337 was here
```
Sticking a blank copyright identifier in this file should make gh actions happy

Reply:
Thank you <3